src/helpers/comment.ts

```diff
@@ -0,0 +1,73 @@
+import { fromMarkdown } from "mdast-util-from-markdown";
+import { gfmFromMarkdown } from "mdast-util-gfm";
+import { gfm } from "micromark-extension-gfm";
+
+export const MarkdownItem = {
```

Comment:
We should also add an image node type here

Reply:
https://github.com/ubiquity/ubiquibot/pull/305/commits/5a466cf96906ed0161567cdd520a7f2ccefd1087